helm/charts/oathkeeper/values.yaml

```diff
@@ -114,3 +114,7 @@ deployment:
   annotations: {}
 # Configure node affinity
 affinity: {}
+
+# Configures controller setup
+oathkeeper-maester:
```

Comment:
I think helm convention is camelCase

Reply:
[Naming convention](https://helm.sh/docs/chart_best_practices/#chart-names) names with `-` are fine too, as long as the chart resided in a dir with the same name :)